ipa_3d_fov_visualization/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>ipa_3d_fov_visualization</name>
+  <version>0.6.12</version>
+  <description>The ipa_3d_fov_visualization package</description>
+
+  <maintainer email="florenz.graf@ipa.fraunhofer.de">Florenz Graf</maintainer>
+  <author>Florenz Graf</author>
+
+  <license>Fraunhofer IPA</license>
```

Comment:
all licenses should be `Apache2`...see any other package
as you already use the proper license header